src/pages/elements/color/tokens.mdx

```diff
@@ -8,6 +8,83 @@ tabs: ['Overview', 'Usage', 'Tokens', 'Code']
 
 import ColorTokenTable from 'components/ColorTokenTable';
 
-## Tokens by theme
+<PageDescription>
 
-<ColorTokenTable />
+Tokens are a method of applying color in a consistent, reusable, and scalable
+way. They are used in place of hard coded values, like hex codes.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Core Tokens</AnchorLink>
+<AnchorLink>Component Tokens</AnchorLink>
+<AnchorLink>AI Tokens</AnchorLink>
+
+</AnchorLinks>
+
+## Core Tokens
+
+Core tokens are global colors that are used across components. They are named
+and grouped by the common UI element that they are applied to.
+
+<AnchorLinks>
+
+<AnchorLink>Background</AnchorLink>
+<AnchorLink>Layer</AnchorLink>
+<AnchorLink>Layer accent</AnchorLink>
+<AnchorLink>Field</AnchorLink>
+<AnchorLink>Border</AnchorLink>
+<AnchorLink>Text</AnchorLink>
+<AnchorLink>Link</AnchorLink>
+<AnchorLink>Icon</AnchorLink>
+<AnchorLink>Support</AnchorLink>
+<AnchorLink>Focus</AnchorLink>
+<AnchorLink>Miscellaneous</AnchorLink>
+
+</AnchorLinks>
+
+<ColorTokenTable type="core" />
+
+## Component Tokens
+
+Some components have their own specific color tokens, known as component tokens.
+They represent the properties associated with a particular component. They are
+not global tokens like the core tokens and should never be used for anything
+other than their own component. See the
+
```

Comment:
There is a weird gap here 
<img width="781" alt="image" src="https://github.com/user-attachments/assets/64d03ad5-ac10-4e71-8bba-eeecc49ac4b2">